lib/runtime-c-api/src/lib.rs

```diff
@@ -451,21 +449,21 @@ pub unsafe extern "C" fn wasmer_module_instantiate(
 
         let namespace = namespaces
             .entry(module_name)
-            .or_insert_with(|| Namespace::new());
+            .or_insert_with(Namespace::new);
```

Comment:
Passing the function name directly instead of a closure saves work for the optimizer.

Reply:
👍